### PR TITLE
Revamp - version 2.0.0

### PR DESCRIPTION
**Pull request by zmrhaljiri**
- New "Activate" toggle switcher
- New increase, decrease, and reset buttons
- Long-press event on increase/decrease buttons
- CSS updates of both light and dark modes, new icons
- Code cleanup (layout, styles, renaming, fix typos)
- High contrast mode support
- Reduced motion support

### CODE REVIEW

File: package-lock.json
Comment (dsactum) [inline package-lock.json:1]: pls remove in favour of yarn

File: helpers/decimals.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const roundToDecimalPlaces = (number, decimalPlaces) => {
```
Comment (dsactum) [inline helpers/decimals.ts]: consider adding come comments with explanation, for example using this scheme https://tsdoc.org

**Comment by dsactum** [comment]
LGTM, please resolve comments and you can also consider squashing commits to create brief and streamlined commit history

File: package-lock.json
Comment (zmrhaljiri) [reply package-lock.json:1]: Thanks, overlooked this.

File: helpers/decimals.ts (hunk 1 above)
Comment (zmrhaljiri) [reply helpers/decimals.ts]: Sure, a good idea, added.

**Review by dsactum — APPROVED** [review]
LGTM

**Pull request merged**